interface/patient_file/report/patient_report.php

```diff
@@ -386,6 +387,11 @@ function show_date_fun(){
                                                 " value='" . $result{"encounter"} . "'" .
                                                 " class='encounter_form' ".
                                                 ">" . xl_form_title($result{"form_name"}) . "<br>\n");
+		*/
+		 echo "<input type='checkbox' name='".$result['formdir']."_".$result['form_id']."'".
```

Comment:
This could be an isolated case. 
I have the latest code dev 5.0.0 running and I added multiple encounters to a patient chart and all forms show even the first encounter forms show as suppose to be shown. 

I would suggest closing this pull request. It can be revisited of this happens for anyone else. 
In my opinion.  (too many court shows)